src/get-language-detector.ts

```diff
@@ -0,0 +1,7 @@
+export function getLangPref(): any {
+  var language =
+    (navigator.languages && navigator.languages[0]) || navigator.language;
+
+  if (language == "ja") return "ja";
+  return "en";
```

Comment:
non-blocking: could just write `return language === "ja" : language : "en"`

please use === instead of ==